### ISSUE TITLE
Block menu should be list + listitems

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2118]
### Subject of the issue
Presenting menu items as a list is easier to digest with a screen reader.

**Comment by oliverfoster** [comment]
merged
